### PR TITLE
Added more youtubers

### PR DESCRIPTION
**Pull request by fluentmoheshwar**
Added Programming Hero, Jhankar Mahbub, Programming Hero Community, Programming Shikhbo

### CODE REVIEW

**Review by Adarsh-jaiss — APPROVED** [review]

**Review by Adarsh-jaiss — APPROVED** [review]

**Review by Adarsh-jaiss — APPROVED** [review]

**Review by Adarsh-jaiss — APPROVED** [review]

**Comment by Adarsh-jaiss** [comment]
-- @KendallDoesCoding  review it ASAP!

**Comment by KendallDoesCoding** [comment]
@Adarsh-jaiss , how much you approved man.

**Comment by KendallDoesCoding** [comment]
If you were fine, you could've bypassed the merge without requiring requirements to be met and merged, didn't have to approve so much lol

File: README.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ If you're a YouTuber reading this and if you think you make beginner-friendly tu
 - [ProgrammingWithHarry](https://www.youtube.com/programmingwithharry) - Makes content around various **programming languages** and has many discussion videos around **coding** and **tech**.
 - [Fireship](https://www.youtube.com/c/Fireship) - Makes content around multiple different programming languages and concepts such as **Carbon** and **Web Development**. He also makes 'explained in 100 seconds' format videos of multiple programming languages.
 - [The Net Ninja](https://www.youtube.com/TheNetNinja) - Makes content around **Web Development** but mostly been doing **Node.js** recently.
+- [Programming Hero](https://www.youtube.com/c/ProgrammingHero) - This channel is weird. This channel will give you project ideas, how to learn something such as **Web Development** or **Game Development** and some motivation.
```
Comment (KendallDoesCoding) [inline README.md:37]:
```suggestion
- [Programming Hero](https://www.youtube.com/c/ProgrammingHero) - This channel will give you project ideas, how to learn something such as **Web Development** or **Game Development** and some motivation.
```

remove "This channel is weird", no content is weird.

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KendallDoesCoding) [inline README.md:37]: Can we inculde This channel makes content around to follow the theme of the other sentences?

**Comment by KendallDoesCoding** [comment]
@fluentmoheshwar Please commit requested changes at the earliest.

File: README.md (hunk 1 above)
Comment (AnimeshKumar923) [reply README.md:37]: If you want uniformity, you can go with "This channel makes content around..."

Comment (KendallDoesCoding) [reply README.md:37]: > If you want uniformity, you can go with "This channel makes content around..."

Yes but it won't work if it's, This channel make's content around project ideas.

@fluentmoheshwar I suggest to make it as follows:

```
This channel makes content around **Web Development** and **Game Development**, they also give motivation and project ideas.

```

What do you think @AnimeshKumar923

Comment (AnimeshKumar923) [reply README.md:37]: Looks good 👍🏻

Comment (AnimeshKumar923) [reply README.md:37]: Also remove the wierd word @KendallDoesCoding

Comment (KendallDoesCoding) [reply README.md:37]: > Also remove the wierd word, no prejudices. @KendallDoesCoding

Yes, I have created a suggestion in my review to do so (scroll up)

Comment (AnimeshKumar923) [reply README.md:37]: Ah, ok, that's good.

**Comment by fluentmoheshwar** [comment]
@KendallDoesCoding closed due to merge conflict

**Comment by fluentmoheshwar** [comment]
@KendallDoesCoding check #48

**Pull request closed without merging**